### PR TITLE
Lesson5 solution

### PR DESCRIPTION
**Pull request by Ognessa**

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shadwork) [inline .idea/gradle.xml:1]: all files from idea folder should be removed from repo

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.onix.internship">
 
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
```
Comment (shadwork) [inline app/src/main/AndroidManifest.xml:6]: application not needed this permissions

**Review by shadwork — APPROVED** [review]
good

File: .idea/gradle.xml (hunk 1 above)
Comment (Ognessa) [reply .idea/gradle.xml]: files removed

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (Ognessa) [reply app/src/main/AndroidManifest.xml]: permissions deleted

**Review by shadwork — APPROVED** [review]

**Pull request merged**